### PR TITLE
First time setup screen

### PR DESCRIPTION
**Pull request by joaoalves03**
After logging in for the first time:
Peek 2024-01-30 22-05

### CODE REVIEW

File: lib/ui/views/firstTime/notifications.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          RichText(
+            text: TextSpan(
+              style: DefaultTextStyle.of(context).style,
+              children: const <TextSpan>[
+                TextSpan(text: 'Nota: ', style: TextStyle(fontWeight: FontWeight.bold)),
+                TextSpan(text: 'O sistema de notificações ainda não está completo, pode não funcionar corretamente'),
+              ],
+            ),
+            textAlign: TextAlign.center
+          ),
```
Comment (rodrigosa7) [inline lib/ui/views/firstTime/notifications.dart:30]: I would add more horizontal padding here. The last sentence gets to close to the screen edge, from what i can see in your printscreen. Its a non blocker, jsut a better UX/UI

File: lib/ui/views/firstTime/theme.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  const FirstTimeThemeSettingsView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            "Customiza a tua app",
```
Comment (rodrigosa7) [inline lib/ui/views/firstTime/theme.dart:14]: Customiza doesn't sound right in my opinion
```suggestion
            "Personaliza a tua app",
```

File: lib/ui/views/first_time.dart [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          if(currentPageIndex == _pages.length - 1) {
+            setFirstTimeAsSeen(true);
+            Navigator.pushReplacement(context,
+                MaterialPageRoute(builder: (context) => const IndexView()));
+          } else {
+            currentPageIndex++;
+          }
+        }); },
+        child: const Icon(Icons.navigate_next),
+      ),
```
Comment (rodrigosa7) [inline lib/ui/views/first_time.dart:46]: We don't have a back button when switching when setting the preferences. What happens if the user wants to change the color scheme? 
We do have a fall back that is going to settings, but would be good to add a back button. 

Style: The button doesn't look that good compared to the rest of the UI of the app, can we try to find a better one?

**Review by rodrigosa7 — CHANGES_REQUESTED** [review]
The code looks okay. I suggested a few changes, if you see that they are not attainable let me know and I would approve the PR since it's nothing that blocking

**Comment by joaoalves03** [comment]
Updated version
Peek 2024-02-01 21-28

File: lib/ui/views/firstTime/notifications.dart (hunk 1 above)
Comment (joaoalves03) [reply lib/ui/views/firstTime/notifications.dart]: Fixed

File: lib/ui/views/first_time.dart [hunk 4]
```diff
@@ -32,23 +32,53 @@ class _FirstTimeViewState extends State<FirstTimeView> {
...
+                        currentPageIndex++;
+                      }
+                    });
+                  },
+                  child: const Icon(Icons.navigate_next),
+                ),
+              ),
+            )
+          ],
+        ),
```
Comment (rodrigosa7) [inline lib/ui/views/first_time.dart:77]: Is there any reason for this isn't wrapped in a Padding() widget instead of repeating both for each button?

**Review by rodrigosa7 — CHANGES_REQUESTED** [review]

File: lib/ui/views/first_time.dart (hunk 4 above)
Comment (joaoalves03) [reply lib/ui/views/first_time.dart:77]: Good catch

**Review by rodrigosa7 — APPROVED** [review]
That's poggers good work, 👍

**Pull request merged**